### DISCUSSION TITLE
Floris first simulation

### DISCUSSION THREAD

**Discussion by Ooussamaa** [NatLabRockies/floris#614] [Q&A] [answered]
Hello,

I am a new user of Floris and Python, I want to simulate and compare the AEP for the four wake models exist in Floris on a specific wind farm layout (contains 33 turbines).

Can you please help me to build an executif python file to run my simulation ( I have already the CSV file for my layout and my wind rose),I tryed to change  the tutorial code but I have not managed to do it 

Thank you

**Comment by Bartdoekemeijer** [comment, marked as answer]
Hi @Ooussamaa,

Welcome to the FLORIS discussion board! The examples contain most of the functionalities you are looking for, including AEP calculations for different wake models. I suggest you take some time to learn basic Python, in case this is unfamiliar to you, and try to run all the examples successfully. By then, you should be able to build up the script that meets your needs. If you then have specific questions, we'd be happy to help or push you in the right direction.

Cheers,
Bart
